extension/custom.css

```diff
@@ -24,6 +24,23 @@ tt,
 .gollum-editor .expanded textarea,
 .gollum-editor .gollum-editor-body,
 .input-monospace,
-.wiki-wrapper .wiki-history .commit-meta code {
+.wiki-wrapper .wiki-history .commit-meta code
+/* +/- pseudo elements on diffs */
+td.blob-code.blob-code-addition:before,
+td.blob-code.blob-code-deletion:before {
```

Comment:
It has to be monospaced at least. The symbols are not the same width in all fonts.